csvutil/csv.go

```diff
@@ -172,21 +210,9 @@ func (tbl *Table) WriteRow(args ...interface{}) error {
 func (tbl *Table) write(args ...interface{}) error {
 	rec := make([]string, len(args))
```

Comment:
we could create a `strings.Builder` variable here, call `.Reset()` just after the `for i, arg := range args {` line and pass that builder (by pointer) to `formatValue`.
`formatValue` would thus only return an `error` (or `nil`)

but ok, it could be part of a subsequent PR, if you prefer.

Reply:
Ok, I changed it in this PR.